pennylane/transforms/condition.py

```diff
@@ -73,13 +94,17 @@ def cond(condition, true_fn, false_fn=None):
             apply if ``condition`` is ``True``
         false_fn (callable): The quantum function of PennyLane operation to
             apply if ``condition`` is ``False``
+        elifs (List(Tuple(bool, callable))): A list of (bool, elif_fn) clauses only when
+            they used with :func:`~.qjit`

```

Comment:
```suggestion
        elifs (List(Tuple(bool, callable))): A list of (bool, elif_fn) clauses. Can only
            be used when with QNodes decorated by :func:`~.qjit`.
```

Reply:
This also works outside QNodes too when it's called with `qml.qjit`.

Reply:
Feel free to update the wording, I just didn't think the initial wording made sense (it needed a comma or a new sentence)